wormhole-connect/src/config/devnet/chains.ts

```diff
@@ -12,40 +12,6 @@ export const DEVNET_CHAINS: ChainsConfig = {
     gasToken: 'ETH',
     chainId: 1,
     icon: Icon.ETH,
-    automaticRelayer: false,
-    maxBlockSearch: 0,
-  },
-  Osmosis: {
```

Comment:
why are you removing these chains?

Reply:
They're not supported so I decided to remove them rather than comment them out. I could add them back if you want

Reply:
added them back